libservice/headers/service/NetlinkCalls.h

```diff
@@ -41,7 +42,13 @@ class NetlinkCalls {
 public:
 	virtual ~NetlinkCalls() = default;
 
+	struct Ipv6Interface {
```

Comment:
Name of struct is misleading. Please change "Interface" suffix to e.g. Ipv6Network